main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
+		self.gjson = json.loads(requests.get("https://ci.appveyor.com/api/projects/" + user + "/" + repo + "/branch/" + branch).text)
+
+	def GetJobs(self):
+		return self.gjson["build"]["jobs"]
+
+	def GetRelease(self, id):
+		idjson = json.loads(requests.get("https://ci.appveyor.com/api/buildjobs/" + str(id) + "/artifacts").text)
+		return idjson
+
+class Updater(object):
+    def __init__(self, settings):
+        self.gsettings = settings
+        appveyor = AppVeyor("xqemu", "master", "xqemu-bot")
+        self.jobs = appveyor.GetJobs()
+        self.release = appveyor.GetRelease(self.jobs[0]["jobId"])
+
+    def update(self):
+        dir = os.path.dirname(os.path.abspath(self.gsettings.settings["xqemu_path"]))
+        if os.path.isdir(dir):
+            fileName = "xqemu.zip"
```

Comment:
If this is part of the appveyor URL, it should probably be part of the AppVeyor constructor?

Reply:
yes it should, i will move it into that.